projects/demo/src/modules/experimental/icon/examples/import/insert-template.md

```diff
@@ -0,0 +1,3 @@
+```html
+<tui-icon>"Batman"</tui-icon>
```

Comment:
Why as `ng-content` ?
Should it be `<tui-icon icon="Batman"></tui-icon>` ?

Reply:
Fixed